tools/validations/dependencyPeerAsDev.js

```diff
@@ -0,0 +1,32 @@
+// Go through peerDependencies declared in package.json file and check that corresponding
+// entries exist in devDependencies. This is to ensure that development environment has all
+// the necessary modules installed in the node_modules directory.
+
+const chalk = require('chalk');
```

Comment:
I think this validation can also help in PF React. It would be nice to ensure `patternfly-react` and `patternfly` and `patternfly-next` dependencies are all in sync on the monorepo. I will create a follow up issue on this.